src/test/crypto_tests.cpp

```diff
@@ -21,7 +21,6 @@
 #include <vector>
 
 #include <boost/test/unit_test.hpp>
-#include <openssl/aes.h>
 #include <openssl/evp.h>
```

Comment:
upstream dropped evp.h should we not? (I presume it may be used, but want some clarity)

Reply:
In our src/test/crypto_tests.cpp at BOOST_AUTO_TEST_CASE(pbkdf2_hmac_sha512_test) we have EVP_sha512  requiring evp.h. upstream doesn't have this testcase